### PR TITLE
fix(@schematics/angular): addProviderToModule bug: add providers into inner Object

### PR DESCRIPTION
**Pull request by bsdfzzzy**
Fix bug when add providers into module.

Fixes: #13080

### CODE REVIEW

File: packages/schematics/angular/utility/ast-utils.ts [hunk 1]
```diff
@@ -449,16 +449,14 @@ export function addSymbolToNgModuleMetadata(
...
-      toInsert = `  ${metadataField}: [${symbolName}]\n`;
+      toInsert = `  ${symbolName}\n`;
     } else {
-      node = expr.properties[expr.properties.length - 1];
-      position = node.getEnd();
       // Get the indentation of the last element, if any.
       const text = node.getFullText(source);
       if (text.match(/^\r?\r?\n/)) {
-        toInsert = `,${text.match(/^\r?\n\s*/)[0]}${metadataField}: [${symbolName}]`;
+        toInsert = `,${(text.match(/^\r?\n\s*/))[0]}${symbolName}`;
```
Comment (alan-agius4) [inline packages/schematics/angular/utility/ast-utils.ts]: The newly brackets here are redundant.

**Comment by bsdfzzzy** [comment]
@alan-agius4 Thanks for checking! I adopted your suggestions and updated this pr.

**Comment by alan-agius4** [comment]
@bsdfzzzy can you please fix the commit msg as you have a typo?

Thanks

**Comment by bsdfzzzy** [comment]
@alan-agius4 emmm...Which one please...

**Comment by alan-agius4** [comment]
`with ut` should be `with it`

**Comment by bsdfzzzy** [comment]
@alan-agius4 emmm... I mean Unit Test.

**Comment by bsdfzzzy** [comment]
@alan-agius4 By the way, could you tell me when can this issue be solved if you know?

**Pull request merged**